packages/synthetix-main/test/integration/modules/core/CollateralConfigurationModule/CollateralConfigurationModule.config.test.ts

```diff
@@ -164,5 +152,25 @@ describe('CollateralModule', function () {
         });
       });
     });
+
+    describe('when a regular user attempts to add a collateral', function () {
+      it('reverts', async () => {
+        const dummyAddress = await user1.getAddress();
+
+        await assertRevert(
+          systems().Core.connect(user1).configureCollateral({
+            tokenAddress: dummyAddress,
+            oracleNodeId: oracleNodeId2,
+            issuanceRatioD18: 400,
+            liquidationRatioD18: 200,
+            liquidationRewardD18: 0,
+            minDelegationD18: 0,
+            depositingEnabled: false,
+          }),
+          `Unauthorized("${await user1.getAddress()}")`,
+          systems().Core
+        );
+      });
+    });
```

Comment:
should we add a small test here to verify the collateral can successfully get its price once its configured? i'm sure it works fine but just to make sure